app/models/user.rb

```
@@ -1090,6 +1089,8 @@ def db_size_in_bytes
       end
       retry unless attempts > 1
       Rollbar.report_exception(e)
+      # INFO: we need to return something to avoid 'disabled' return value
+      nil
```

You're right... then better the nil, but we should start wrapping all rollbar calls (using the existing [error notifier I mean](https://github.com/CartoDB/cartodb/blob/master/config/initializers/error_notifier.rb)) so in dev/test they output stuff and we don't silence the errors and by mistake return "disabled"s...